queries.sql

```diff
@@ -55,4 +55,81 @@ animals
  FROM
  animals
  WHERE
- weight_kg >=10.4 AND weight_kg <=17.3;
\ No newline at end of file
+ weight_kg >=10.4 AND weight_kg <=17.3;
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```

Comment:
- [ ] I noticed the screenshot for this query has a commit statement which shouldn't be part of the transaction and is creating a mismatch between your query and the corresponding results, so please take a screenshot with the proper query.
  ![image](https://github.com/kkmanuu/vet_clinic/assets/104968236/ee1718f2-77f1-4558-8b1d-03d99ba4a6bf)

Reply:
@AbelG101   I'm done